keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifact/UnsupportedArtifactTypeException.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.artifact
+
+import com.netflix.spinnaker.keel.api.ArtifactStatus
+
+class UnsupportedArtifactTypeException(
+  val type: String
+) : RuntimeException("Artifact type $type not in supported types: ${ArtifactStatus.values()}")
```

Comment:
This should be `ArtifactType.values()`, no?

Reply:
Yes! Good catch.